public/js/buttonFunctions.js

```diff
@@ -128,7 +151,9 @@ let updateGradePage = function() {
   classesTable.replaceData(currentTableData.currentTermData.classes);
   categoriesTable.setData(currentTableData.currentTermData.classes[selected_class_i].categoryDisplay);
 
-  assignmentsTable.replaceData(currentTableData.currentTermData.classes[selected_class_i].assignments);
+  assignmentsTable.replaceData(currentTableData.currentTermData.classes[selected_class_i].assignments.filter((obj) => {
+    return obj.placeholder !== true
+  }));
```

Comment:
This arrow function can be simplified using [implicit returns][1] (when braces are omitted) and [object destructuring syntax][2] (and brought within 80 characters per line):
```suggestion
  assignmentsTable.replaceData(currentTableData.currentTermData
	  .classes[selected_class_i].assignments
	  .filter(({ placeholder }) => !placeholder)
  );
```

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Basic_syntax
[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment